lib/plugins/bidirectionalData.js

```diff
@@ -19,6 +19,8 @@
  *
```

Comment:
CNR entry for this PR is missing.

Reply:
Fixed in 4232cc6baf524d74560dda9ed23f9c044286954b